includes/class-freemius.php

```diff
@@ -6620,10 +6643,6 @@ private function get_install_sync_cron_blog_id() {
          * @param int $except_blog_id Since 2.0.0 when running in a multisite network environment, the cron execution is consolidated. This param allows excluding excluded specified blog ID from being the cron executor.
          */
         private function schedule_install_sync( $except_blog_id = 0 ) {

```

Comment:
I see that this function is only called from within the `maybe_schedule_install_sync_cron`. So I am suggesting to completely remove this function and move the one line remaining code inside the new function only.

This way we will protect ourselves from accidentally calling this one, which could be harmful as we've seen.